### ISSUE TITLE
NN blitz tutorial: some small improvements

### ISSUE THREAD

**Issue by murphyk** [pytorch/tutorials#533]
I had some small suggestions for 
https://pytorch.org/tutorials/beginner/blitz/neural_networks_tutorial.html

- before the picture of the neural net, mention this is the famous "LeNet" example (and maybe give reference)

- explain that the constructor only contains the layers with adjustable parameters,
and the deterministic layers only appear in forward()

- print(len(params)) gives the answer 10, but there are only 5 parameter blocks. Explain that the bias terms are treated separately.

- "net.zero_grad(); out.backward(torch.randn(1, 10))" does not print anything, which makes one think there is something wrong. How about adding "print(params[1].grad)" and explain this is the gradient of the loss (evalyated at a random target vector) wrt the first layer bias vector.

- The comment about batches should come before the statement "input = torch.randn(1, 1, 32, 32)" otherwise that is unclear
